### DISCUSSION TITLE
How to keep image resource fixed quality

### DISCUSSION THREAD

**Discussion by HusniMuhammad** [LinwoodDev/Butterfly#745] [Q&A] [answered]
eg: export potion of note to 640X640 export to img

**Comment by CodeDoctorDE** [comment]
You have different sliders on the export dialog.
The x and y are the positions.
The final image size is width × quality, height × quality.

If you set your size to 300x300 and quality to 2 the final image will be 600x600 in size

**Reply by HusniMuhammad** [reply]
How to create fixed 300X300 square shape

**Reply by CodeDoctorDE** [reply, marked as answer]
Set quality to 1 and size to 300 width and 300 height. Adjust zoom and position to your liking. You will see a preview on the dialog
